content/en/docs/tasks/observability/logs/telemetry-api/index.md

```diff
@@ -99,8 +99,30 @@ spec:
 EOF
 {{< /text >}}
 
+1. Set default filter access log with CEL expression
+
+The following configuration displays access logs only when the response code is greater or equal to 400 or the request went to the BlackHoleCluster or the PassthroughCluster:
+Note: The xds.cluster_name is only available with Istio release 1.16.2 and higher
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n default -f -
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+name: disable-details-logging
+namespace: istio-system
```

Comment:
`-n default` in L108, so I think you should remove this line.

Reply:
removed the -n default